packages/sw-background-sync-queue/src/lib/queue.js

```diff
@@ -1,6 +1,7 @@
 import stringHash from 'string-hash';
 import idbQHelper from './idbHelper';
 import requestManager from './requestManager';
+import bcmanager from './broadcastManager'; 
 
 let _queue;
 let _config;
```

Comment:
I think Jeff may have mentioned this, but I believe you'll want to switch away from module scoped variables here to member variables instead.